packages/angular/src/generators/application/lib/add-e2e.ts

```diff
@@ -25,6 +29,24 @@ export async function addE2e(tree: Tree, options: NormalizedSchema) {
       skipPackageJson: options.skipPackageJson,
       skipFormat: true,
     });
+  } else if (options.e2eTestRunner === 'playwright') {
+    addProjectConfiguration(tree, options.e2eProjectName, {
```

Comment:
would it make playwright a dev dependency of angular regardless what e2eTestRunner developers choose. for cypress, i saw some places use ensurePackage to import from @nx/cypress library, like:
```
  const { cypressProjectGenerator } = ensurePackage<
    typeof import('@nx/cypress')
  >('@nx/cypress', nxVersion);
```
should we do the same for playwright?

Reply:
good point. I'll refactor to use `ensurePackage`